libmogwai-schedule-client/scheduler.h

```diff
@@ -84,4 +84,13 @@ MwscScheduleEntry *mwsc_scheduler_schedule_finish (MwscScheduler        *self,
                                                    GAsyncResult         *result,
                                                    GError              **error);
 
+void               mwsc_scheduler_schedule_entries_async  (MwscScheduler        *self,
```

Comment:
I am thinking whether it'd be better to call it `mwsc_scheduler_schedule_list_async`.
The reason is that we already have `mwsc_scheduler_schedule_async` which uses `entry` here as something implicit, so the new method should be the same but just indicate that it takes a list instead.

Either way, I am not strongly opposed to the way you wrote it. So if you think we shouldn't change it, I am fine with it.

Reply:
I disagree; I think the fact that `entries` is plural adequately conveys that it takes a list.